src/js/containers/search/filters/naics/NAICSListContainer.jsx

```diff
@@ -0,0 +1,177 @@
+            };
+
+            this.naicsSearchRequest = SearchHelper.fetchNAICS(naicsSearchParams);
+
+            this.naicsSearchRequest.promise
+                .then((res) => {
+                    const data = res.data.results;
+                    let autocompleteData = [];
+                    const search = new Search('naics');
+                    // ordering by prefix if there are matches returned that begin w/ the exact text
+                    search.indexStrategy = new PrefixIndexStrategy();
+                    search.addIndex(['naics']);
+                    search.addIndex(['naics_description']);
+                    search.addDocuments(data);
+                    const results = search.search(this.state.naicsSearchString);
+                    const improvedResults = slice(results, 0, 10);
+
+                    // Filter out any selected NAICS that may be in the result set
+                    const selectedNAICS =
+                    this.props.selectedNAICS.toArray().map((naics) => omit(naics, 'identifier'));
+                    if (improvedResults && improvedResults.length > 0) {
+                        autocompleteData = differenceWith(improvedResults, selectedNAICS, isEqual);
+                    }
+                    else {
+                        autocompleteData = improvedResults;
+                    }
+
+                    this.setState({
+                        noResults: autocompleteData.length === 0
+                    });
+
+                    // Add search results to Redux
+                    this.props.setAutocompleteNAICS(autocompleteData);
+                })
+                .catch((err) => {
+                    if (!isCancel(err)) {
+                        this.setState({
+                            noResults: true
+                        });
+                    }
+                });
+        }
+        else if (this.naicsSearchRequest) {
+            // A request is currently in-flight, cancel it
+            this.naicsSearchRequest.cancel();
+        }
+    }
+
+    clearAutocompleteSuggestions() {
+        this.props.setAutocompleteNAICS([]);
+    }
+
+    handleTextInput(naicsInput) {
+        // Clear existing NAICS to ensure user can't select an old or existing one
+        this.props.setAutocompleteNAICS([]);
+
+        // Grab input, clear any exiting timeout
+        const input = naicsInput.target.value;
+        window.clearTimeout(this.timeout);
+
+        // Perform search if user doesn't type again for 300ms
+        this.timeout = window.setTimeout(() => {
+            this.queryAutocompleteNAICS(input);
+        }, 300);
+    }
+
+    render() {
+        return (
+            <Autocomplete
+                {...this.props}
+                values={this.state.autocompleteNAICS}
+                handleTextInput={this.handleTextInput}
+                onSelect={this.props.selectNAICS}
+                placeholder="eg: 33641 - Aircraft Manufacturing"
```

Comment:
Also can you change the NAICS to `336411` which is the correct value for Aircraft Manufacturing?